### PR TITLE
Add blog tab to demo site

### PR DESCRIPTION
**Pull request by Pessimistress**

### CODE REVIEW

File: demo/src/components/blog.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/* global window */
+import React, {Component, PropTypes} from 'react';
+import autobind from 'autobind-decorator';
+
+export default class Blog extends Component {
+
+  componentWillReceiveProps(nextProps) {
+    const {route} = nextProps;
+    if (this.props.route !== route) {
```
Comment (ibgreen) [inline demo/src/components/blog.js]: To capture the logic better, why not simply write 
```this.nextProps.route !== this.props.route```

File: demo/src/components/blog.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+/* global window */
+import React, {Component, PropTypes} from 'react';
+import autobind from 'autobind-decorator';
+
```
Comment (ibgreen) [inline demo/src/components/blog.js]: I think it is worth declaring the propTypes. It really helps when reading the code.

File: demo/webpack/config.js [hunk 3]
```diff
@@ -35,6 +35,7 @@ module.exports = {
     modules: [resolve(rootDir, 'node_modules'), resolve(demoDir, 'node_modules')],
     alias: {
       'deck.gl': libSources,
+      'd3-scale': resolve('./node_modules/d3-scale'),
```
Comment (ibgreen) [inline demo/webpack/config.js]: Any insight into why is this not covered by the modules statement above?

File: demo/src/components/blog.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      const href = anchor.getAttribute('href');
+      if (href.indexOf('/blog/') === 0) {
+        anchor.href = `${pathname}#${href}`;
+        anchor.target = '_parent';
+      }
+    }
+  }
+
+  render() {
+    let {routeParams: {splat = 'index'}} = this.props;
```
Comment (ibgreen) [inline demo/src/components/blog.js]: Why `splat`? Add a comment?

File: demo/src/components/blog.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+  componentWillReceiveProps(nextProps) {
+    const {route} = nextProps;
+    if (this.props.route !== route) {
+    }
+  }
+
+  // redirect links to parent window
+  @autobind _onPostLoaded(evt) {
+    const doc = evt.target.contentDocument;
```
Comment (ibgreen) [inline demo/src/components/blog.js]: Would be great with more explanation here. If this is building on some github functionality, a link to the relevant docs. Can't say I understand how this works.

File: demo/src/components/blog.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  render() {
+    let {routeParams: {splat = 'index'}} = this.props;
+
+    splat = splat.replace(/\.html$/, '');
+
+    return (
+      <div className="blog-page">
+        <iframe src={`./blog/${splat}.html`} onLoad={ this._onPostLoaded } />
```
Comment (ibgreen) [inline demo/src/components/blog.js]: Sorry - what is this iframe encapsulating? What is being served from `./blog`?

Comment (Pessimistress) [inline demo/src/components/blog.js]: `./blog` contains the static site generated by Jekyll from the `blog` branch.

**Pull request closed without merging**